<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>231</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>75</width>
     <height>150</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>75</width>
     <height>150</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="pixmap">
    <pixmap>Graphics/hydrococo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>180</y>
     <width>100</width>
     <height>150</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>150</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Graphics/indomilk-removebg-preview.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>140</y>
     <width>231</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>370</y>
     <width>231</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>400</y>
     <width>90</width>
     <height>150</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>90</width>
     <height>150</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Graphics/pocky.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>370</y>
     <width>231</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>430</y>
     <width>125</width>
     <height>100</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>125</width>
     <height>100</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Graphics/nabati-removebg-preview.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelOctoVendingMachine">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Terminal</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  bold 15pt &quot;Terminal&quot;;</string>
   </property>
   <property name="text">
    <string>OCTO VENDING MACHINE KELOMPOK 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="keMain2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>250</y>
     <width>31</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>190</y>
     <width>95</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_1">
      <property name="styleSheet">
       <string notr="true">font: 75 8pt &quot;System&quot;;</string>
      </property>
      <property name="text">
       <string>Hydrococo</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="quantity_1">
      <property name="styleSheet">
       <string notr="true">font: 75 bold 8pt &quot;System&quot;;</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="harga_1">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: bold 8pt &quot;NSimSun&quot;;</string>
      </property>
      <property name="text">
       <string>Rp7000</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonBeli_1">
      <property name="text">
       <string>Beli</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>190</y>
     <width>95</width>
     <height>127</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 8pt &quot;System&quot;;</string>
      </property>
      <property name="text">
       <string>Indomilk</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="quantity_2">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 bold 8pt &quot;System&quot;;</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="harga_2">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: bold 8pt &quot;NSimSun&quot;;</string>
      </property>
      <property name="text">
       <string>Rp5000</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonBeli_2">
      <property name="text">
       <string>Beli</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>410</y>
     <width>95</width>
     <height>127</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">font: 75 8pt &quot;System&quot;;</string>
      </property>
      <property name="text">
       <string>Pocky</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="quantity_3">
      <property name="styleSheet">
       <string notr="true">font: 75 bold 8pt &quot;System&quot;;</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="harga_3">
      <property name="styleSheet">
       <string notr="true">font: bold 8pt &quot;NSimSun&quot;;</string>
      </property>
      <property name="text">
       <string>6000</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonBeli_3">
      <property name="text">
       <string>Beli</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>410</y>
     <width>95</width>
     <height>127</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">font: 75 8pt &quot;System&quot;;</string>
      </property>
      <property name="text">
       <string>Nabati</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="quantity_4">
      <property name="styleSheet">
       <string notr="true">font: 75 bold 8pt &quot;System&quot;;</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="harga_4">
      <property name="styleSheet">
       <string notr="true">font: bold 8pt &quot;NSimSun&quot;;</string>
      </property>
      <property name="text">
       <string>3000</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonBeli_4">
      <property name="text">
       <string>Beli</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>tableWidget_2</zorder>
  <zorder>tableWidget</zorder>
  <zorder>label</zorder>
  <zorder>label_7</zorder>
  <zorder>tableWidget_3</zorder>
  <zorder>label_8</zorder>
  <zorder>tableWidget_4</zorder>
  <zorder>label_9</zorder>
  <zorder>labelOctoVendingMachine</zorder>
  <zorder>keMain2</zorder>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>verticalLayoutWidget_2</zorder>
  <zorder>verticalLayoutWidget_3</zorder>
  <zorder>verticalLayoutWidget_4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
